.github/workflows/test.yml

```diff
@@ -31,6 +31,12 @@ jobs:
       run:
         shell: bash
     steps:
+    - name: Use XCode 12.0 so pycifrw will build
```

Comment:
Should this be "11.7" instead of "12.0"?

Reply:
Indeed, I tried 12.0 first :-)

Reply:
Fixed